libraries/AP_HAL_ChibiOS/hwdef/scripts/STM32H743xx.py

```diff
@@ -20,8 +20,9 @@
     # flags of 4 means memory can be used for SDMMC DMA
     'RAM_MAP' : [
         (0x20000000, 128, 2), # DTCM, tightly coupled, no DMA, fast
-        (0x30000000, 288, 0), # SRAM1, SRAM2, SRAM3
-        (0x38000000,  64, 1), # SRAM4. This supports both DMA and BDMA ops
+        (0x30000000, 272, 0), # SRAM1, SRAM2
+        (0x30040000,  16, 1), # SRAM3. This supports both DMA and BDMA ops.
```

Comment:
should move this to the last region, so it isn't swallowed by other non-DMA ops

Reply:
Fixed